public/jira/derived/work-timing/work-timing.ts

```diff
@@ -78,7 +78,9 @@ export function deriveWorkTiming(
 ): DerivedWorkTiming {
   const isConfidenceValid = isConfidenceValueValid(normalizedIssue.confidence);
 
-  const usedConfidence = isConfidenceValid ? normalizedIssue.confidence! : getDefaultConfidence(normalizedIssue.team);
+  const usedConfidence = isConfidenceValid
+    ? normalizedIssue.confidence!
```

Comment:
a trailing `!` is very convenient, but _very_ easy to miss when skimming. It is telling typescript "no, you're wrong. I know I told you it might be undefined before, but actually its not". While there _are_ situations where that statement is accurate, I like to be more explicit so it jumps out when skimming. If the type is `number | null | undefined`, then I'd put `normalizedIssue.confidence as string` instead of `normalizedIssue.confidence!`.